runtime/parser/keyword.go

```diff
@@ -62,4 +62,10 @@ const (
 	keywordSwitch      = "switch"
 	keywordDefault     = "default"
 	keywordEnum        = "enum"
+	keywordExtend      = "extend"
```

Comment:
Is there a doc with info about these new keywords? Can't find anything in the FLIP or related issues.

Reply:
The new keywords are all the ones mentioned in the FLIP here: https://github.com/onflow/flow/pull/1101 under the headings for creating extensions, attaching extensions and removing extensions.